index.css

```diff
@@ -1,87 +1,392 @@
+    border-color: var(--danger);
+}
+.modal-btn{
+    color: whitesmoke;
+    background-color: var(--primarycolor);
+    padding: 20px;
+    border-style: none;
+    border-radius:5px;
+    text-align: center;
+    cursor: pointer;
+    width: 20%; 
+     font-size: 25px; 
+}
+.modal-header{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    vertical-align: centre;
+}
+.modal{
+    border: var(--d-grey) 1px solid;
+    width: 50%;
+    border-radius: 2px;
+}
+.modal-title{
+    margin: 10px 0 0 10px;
+    font-size: 1.5rem;
+}
+.close-icon{
+    height: 13px;
+    width: 13px;
+    position: relative;
+    right: 20px;
+    top: 12px;
+}
+.buttons{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;    
+}
+.modal-body{
+    margin: 2px 10px;
+    padding: 4px 0;
+}
+hr{
+    border: solid 0.1px var(--light-grey);
+}
+.badge{
+    padding-right: 0.3rem;
+    padding-left: 0.1rem;
+    border: 1px white solid;
+    margin: 0.3rem;
+    background-color: #fff;
+    color: var(--primarycolor);
+    border-radius: 50%;
+    text-align: center;
+}
+.online-offline{
+    height: 0.5rem;
+    width: 0.5rem;
+    position: relative;
+    right: 2.1rem;
+    top: -1rem;
+}
+.cart{
+    height: 2.5rem;
+    width: 2.5rem;
+}
+.cart-badge{
+    position: relative;
+    bottom: 2rem;
+    right: 1rem;
+    padding-right: 0.3rem;
+    padding-left: 0.1rem;
+    border: 1px white solid;
+    margin: 0.3rem;
+    background-color:var(--primary);
+    color: white;
+    border-radius: 50%;
+    text-align: center;
+}
+.card-img{
+    height: 12rem;
+}
+.card{
+    border: black 1px solid;
+    margin: 1rem;
+    width: auto;
+    text-align: center;
+    height: fit-content;
+}
+.card-data {
+    margin: 0.8rem;
+  }
+.card-div{
+    display: flex;
+    flex-wrap: wrap;
+}
+.category-card {
+    margin: 2rem;
+    border: 8px solid grey;
+    height: auto;
+    width: 20%;
+  }
+.card-badge{
+    position: relative;
+    top: 1.7rem;
+    right: 3.5rem;
+    padding-right: 0.3rem;
+    padding-left: 0.1rem;
+    border: 1px white solid;
+    margin: 0.3rem;
+    background-color:palevioletred;
+    color: white;
+    text-align: center;
+}
+.tooltip {
+    position: relative;
+    display: inline-block;
+    margin-left: 3rem;
+}
+  
+.tooltip .tooltiptext {
+    visibility: hidden;
+    width: 120px;
+    background-color: var(--primarycolor);
+    color: #fff;
+    text-align: center;
+    border-radius: 6px;
+    padding: 5px 0;
+    position: absolute;
+    z-index: 1;
+    bottom: 80%;
+    left: 90%;
+    margin-left: -50px;
+}
+  
+.tooltip .tooltiptext::after {
+    content: "";
+    position: absolute;
+    top: 100%;
+    left: 50%;
+    margin-left: -5px;
+    border-width: 5px;
+    border-style: solid;
+    border-color: var(--primarycolor) transparent transparent transparent;
+}
+  
+.tooltip:hover .tooltiptext {
+    visibility: visible;
+    transition-duration: 0.05s;
+}
+#toast{
```

Comment:
if your are performing js on it i would suggest to not use id instead you can use class so on that you can do getElementByClassName